index.js

```diff
@@ -182,3 +182,9 @@ Object.defineProperty(module.exports, 'videoCodecs', {
     return codecs;
   }
 });
+
+Object.defineProperty(module.exports, 'supportsHevcHardwareEncoding', {
+  get() {
+    return supportsHevcHardwareEncoding;
+  }
+});
```

Comment:
Why not just `module.exports.supportsHevcHardwareEncoding = supportsHevcHardwareEncoding`?

And I think it should be documented so other can take advantage of it too.

Reply:
😅 thought it was a function call initially, forgot to change it.

Should be fixed and added a readme entry as well